cl/opinion_page/templates/docket_tabs.html

```diff
@@ -88,7 +88,7 @@ <h2>{{ docket.court }}</h2>
     {% if docket.pacer_docket_url or docket_entries %}
       <div class="v-offset-above-2">
         {% if docket_entries %}
-          <a href="/?type=r&amp;q=docket_id%3A{{ docket.pk }}"
```

Comment:
this ... from my limited understanding is not what was intended .

Reply:
The idea here is that we don't want Google spending resources crawling our search result pages, since they're not very structured or useful.